### ISSUE TITLE
Many servers failing during genconfig

### ISSUE THREAD

**Issue by myleshorton** [getlantern/lantern#2341]
Most common error appears to be:

```
genconfig: Skipping fallback 178.62.213.97 because dialing Google failed: Unable to dial server: EOF
```

**Comment by myleshorton** [comment]
Maybe you could give this a look during the week this week @aranhoide?

**Comment by aranhoide** [comment]
There was a mismatch between the ciphers accepted by some fallbacks and the ones accepted by Lantern 2.  I *think* this happened when `-keyalg` was chosen to be `DSA` (which happened with probability 2/7).  Removing that option [1] fixed all fallbacks.  The config in the following PR contains the certs corresponding to the fixed config:

https://github.com/getlantern/flashlight-build/pull/152

[1] https://github.com/getlantern/lantern_aws/commit/93f8fa5e67ca577ba7ab6dfc7a4009df083fa888

**Comment by myleshorton** [comment]
You're the man thanks will look at it.

**Comment by aranhoide** [comment]
Since this one is strictly better for about everyone using Lantern 2.0, I'll merge it and deploy it to all 2.0 clients.

**Comment by aranhoide** [comment]
Merged and uploaded for all 2.0 clients (IIRC; I've uploaded this to `valencia` and `2.0.0-beta3`).
